manifests/resource/mailhost.pp

```diff
@@ -101,6 +103,7 @@
   if ($ssl_cert != undef) {
     validate_string($ssl_cert)
   }
+  validate_string($ssl_protocols)
```

Comment:
I feel blah about having this as a string rather than an array, but I guess there's a precedent.